src/main/resources/application.properties

```diff
@@ -10,6 +10,6 @@ spring.jpa.properties.hibernate.use_sql_comments=true
 spring.jpa.properties.hibernate.type=trace
 
 email.sender=helpmatchapp@gmail.com
-#${EMAIL_SENDER}
 email.password=dtuilyswyudssver
-#${EMAIL_PASSWORD}
\ No newline at end of file
+email.subject=C�digo de Verifica��o
+email.body=
```

Comment:
Se não for usar, pode remover essa linha. Entendi a dificuldade de fragmentar o texto do Body, vou pensar em alguma forma de podermos extrair aquele texto de lá.